src/controllers/fetchTopics.js

```diff
@@ -0,0 +1,6 @@
+import axios from "axios"
```

Comment:
You don't really need to be creating new files for every api function you write, they could be included in the same file (named something like `api.js`) 

If you're not sure about how to export multiple functions from a file using ES6 modules you can take a look at [these notes](https://notes.northcoders.com/courses/js-front-end/react-intro#es6-modules) as a refresher.